Support/ERCS_Logs/ReadMe.md

```diff
@@ -37,5 +37,23 @@ Specifies parameter to filter log collection. Valid formats are comma separated
 ## ErcsName ##
 Specifies privileged endpoint virtual machine name or IP address to use. Example: AzS-ERCS01 or 192.168.200.255
 
+## AzSCredentials ##
+Specifies credentials the script will use to connect to Azure Stack privileged endpoint. Format must be in one of the 2 formats:
+
+- (Get-Credential -Message "Azure Stack Credentials")
+- (Get-Credential)
+
+## ShareCred ##
+Specifies credentials the script will use to build a local share Format must be in one of the 2 formats:
+
+- (Get-Credential -Message "Local Share Credentials" -UserName $env:USERNAME)
+- (Get-Credential)
+
+## InStamp ##
+Specifies if script is running on Azure Stack machine such as Azure Stack Development Kit deployment or DVM.
+
+- Yes
+- No
+
 ## Example Use ##
-	.\ERCS_AzureStackLogs.ps1 -FromDate (get-date).AddHours(-4) -ToDate (get-date) -FilterByRole VirtualMachines,BareMetal -ErcsName AzS-ERCS01
+	.\ERCS_AzureStackLogs.ps1 -FromDate (get-date).AddHours(-4) -ToDate (get-date) -FilterByRole VirtualMachines,BareMetal -ErcsName 192.168.200.225 -AzSCredentials (Get-Credential -Message "Azure Stack Credentials") -ShareCred (get-credential -Message "Local Share Credentials" -UserName $env:USERNAME) -InStamp No

```

Comment:
does the name have to be an IP?  There is no guarantee of IP names for even one node VMs as the code assigns the next available IP in the range.. If an new objects appears before this IP in a future release it will get a new IP.

Why cant it be a name?

Reply:
as the readme said it can be either
See readme:
Specifies privileged endpoint virtual machine name or IP address to use. Example: AzS-ERCS01 or 192.168.200.255